### PR TITLE
Vpn page design update

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Changed layout and text, based on if the user has vpn or not.

### CODE REVIEW

File: android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 public class VPNStatusDetailActivity extends AbstractActivity {
 
     private final OnGetVPNClickListener onGetVPNClickListener;
+    private boolean isProtectedConnection;
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: why? I see the reason, but it shouldn't be that way to start with.

Comment (marcelinkaaa) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: Still a beginner. Didn't know what else to do without changing the structure more. Any better solutions?
Did it so it connects to the public OnGetVPNClickListener

File: android/res/layout/activity_no_btcwallet_available.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    android:background="@color/app_ligh_background" >
+    android:background="@color/app_light_background" >
```
Comment (gubatron) [inline android/res/layout/activity_no_btcwallet_available.xml]: good catch

File: android/res/values/colors.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
 /*
- * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marclinkaaa)
```
Comment (gubatron) [inline android/res/values/colors.xml]: :muscle:

Comment (gubatron) [inline android/res/values/colors.xml]: I think there's a typo on your twitter handle though.

File: android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java [hunk 4]
```diff
@@ -55,20 +56,31 @@ protected void initComponents(Bundle savedInstanceState) {
             getActionBar().setDisplayHomeAsUpEnabled(true);
             getActionBar().setIcon(android.R.color.transparent);
         }
-
         final Intent intent = getIntent();
-        final boolean isProtectedConnection = intent.getAction() != null &&
+        isProtectedConnection = intent.getAction() != null &&
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: you can revert this line, and declare the variable locally here, it's not used elsewhere, therefore no need to make it an instance's property.

File: android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java [hunk 5]
```diff
@@ -116,7 +128,13 @@ public OnGetVPNClickListener(VPNStatusDetailActivity owner) {
 
         @Override
         public void onClick(VPNStatusDetailActivity owner, View v) {
-            UIUtils.openURL(owner, Constants.PIA_URL);
+            if (owner.isProtectedConnection) {
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: oh, I see here...

File: android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java (hunk 4 above)
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java]: don't worry, I'll go ahead and refactor this, will be even cleaner.

File: android/res/drawable/vpn_button_background_selector.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marclinkaaa)
```
Comment (gubatron) [inline android/res/drawable/vpn_button_background_selector.xml]: typo

File: android/res/drawable/vpn_button_background_selector_off.xml [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marclinkaaa)
```
Comment (gubatron) [inline android/res/drawable/vpn_button_background_selector_off.xml]: typo

File: android/res/drawable/vpn_button_background_selector_on.xml [hunk 8]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
 /*
- * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marclinkaaa)
```
Comment (gubatron) [inline android/res/drawable/vpn_button_background_selector_on.xml]: typo

File: android/res/layout/view_vpn_status_detail.xml [hunk 9]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
 /*
- * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), and
+ * Marcelina Knitter (@marclinkaaa)
```
Comment (gubatron) [inline android/res/layout/view_vpn_status_detail.xml]: typo

**Pull request merged**